scripts/transform_doxygen_html.py

```diff
@@ -529,6 +529,22 @@ def parse_header(header_path):
     print("ERROR: ", e, exc_tb.tb_lineno)
   return h_json
 
+def update_release_version(doxyfile_path, site_config_path):
+  try:
+    with open(doxyfile_path) as f:
+      doxy_content = f.read()
+    version_search = re.search("(\nPROJECT_NUMBER\s*=\s*)([\d.]+)", doxy_content)
```

Comment:
The `.` here should probably be a `\.` ?

Reply:
I think all characters inside brackets are treated as literals in a regular expression (except for a handful of exceptions)...? I might be wrong though.

Reply:
Ah, so they are! Please accept my apologies.